ecosystem/typescript/sdk/src/transaction_builder/aptosTypes/index.ts

```diff
@@ -0,0 +1,1292 @@
+import { Serializer, Deserializer, Seq, Tuple, ListTuple, bool, uint8, uint64, uint128, str, bytes } from "../serde";
```

Comment:
wow, nice, might be a good idea to break this up into a few files.

Reply:
This is a quite old commit. Have done a lot of refactoring. This file doesn't exist any more :)